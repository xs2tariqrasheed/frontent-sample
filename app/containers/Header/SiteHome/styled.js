import styled from 'styled-components';
import { Link } from 'react-router-dom';
import theme from '../../../styles/theme';

export const Label = styled(Link)`
  font-size: ${theme.fontSize.larger};
  color: ${theme.color.white};
  text-decoration: none;

  :hover {
    cursor: pointer;
    color: ${theme.color.tin};
  }
`;

export const HomeLogo = styled.div`
  margin-top: 12px;
`;
