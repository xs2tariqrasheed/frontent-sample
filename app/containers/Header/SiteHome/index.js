/*
 * Icon for home page
 *
 * Icon for home page
 *
 */

import React from 'react';
import { Label, HomeLogo } from './styled';
import { BlockleteLogo } from '../../../components/Icons';

export default props => (
  <h1 {...props} className="items-center">
    <HomeLogo>
      <Label to="/">
        <BlockleteLogo />
      </Label>
    </HomeLogo>
  </h1>
);
