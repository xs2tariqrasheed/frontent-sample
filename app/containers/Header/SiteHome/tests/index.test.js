import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import { BrowserRouter } from 'react-router-dom';

import SiteHome from '../index';

describe('<SiteHome />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <BrowserRouter>
        <IntlProvider locale="en">
          <SiteHome />
        </IntlProvider>
      </BrowserRouter>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
