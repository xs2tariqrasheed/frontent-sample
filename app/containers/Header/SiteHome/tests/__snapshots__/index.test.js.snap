// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteHome /> should render and match the snapshot 1`] = `
<h1
  class="items-center"
>
  <a
    class="styled__Label-pedict-0 iEBUsl"
    href="/"
  >
    <svg
      height="56"
      viewBox="0 0 100 56"
      width="100"
      xmlns="http://www.w3.org/2000/svg"
    >
      <defs>
        <lineargradient
          id="d4ban9q7ja"
          x1="50%"
          x2="50%"
          y1="58.611%"
          y2="43.636%"
        >
          <stop
            offset="0%"
            stop-color="#26225D"
          />
          <stop
            offset="100%"
            stop-color="#3A348F"
          />
        </lineargradient>
        <lineargradient
          id="xdra1xeu7b"
          x1="50%"
          x2="50%"
          y1="82.242%"
          y2="-31.453%"
        >
          <stop
            offset="0%"
            stop-color="#FFF"
          />
          <stop
            offset="100%"
            stop-color="#FFC430"
          />
        </lineargradient>
      </defs>
      <g
        fill="none"
      >
        <path
          d="M91.602 20.47h-8.546v-2.896c-.694-.28-1.408-.545-2.127-.804v-2.836l5.158-.921v-.102l-5.158-.922h-.69v4.531c-.703-.244-1.418-.509-2.137-.717v2.296c-5.406-1.78-10.979-3.012-16.633-3.676 1.1-4.722-.816-9.634-4.827-12.37-4.01-2.737-9.294-2.737-13.304 0-4.011 2.736-5.928 7.648-4.828 12.37-5.701.662-11.32 1.9-16.77 3.696v-2.316c-.725.229-1.434.473-2.138.717v-2.586l5.138-.921v-.102l-5.163-.922h-.69v4.776c-.719.264-1.433.509-2.132.81v2.896H8.21L0 47.269l7.357 6.776h21.398l1.98 1.798h38.163l1.98-1.798H92.52l7.44-6.776-8.358-26.798z"
          fill="url(#d4ban9q7ja)"
        />
        <ellipse
          cx="49.98"
          cy="11.801"
          fill="#FFF"
          rx="8.689"
          ry="8.67"
        />
        <path
          d="M41.383 10.452c.103.617.274 1.222.51 1.802-.027.046-.034.094-.035.144v.05c-.034.407.18.793.538.982l.073.035h.036c.062.021.125.036.189.046.418.16.683.573.653 1.018l.002.072-.004.132.002.061c-.024.262.058.523.227.725.17.202.413.327.676.35.069.01.138.013.208.01l.103-.01.092-.024c.092-.018.186-.024.28-.017.448.046.81.379.896.813l.013.083.004.078c.034.362.264.68.603.823.338.142.725.087 1.008-.139l.063-.055.07-.057c.193-.145.433-.214.675-.192.324.03.612.214.778.489l.039.07.033.073c.158.306.466.508.814.529.412.042.804-.176.986-.542l.034-.074.04-.074c.14-.237.373-.406.645-.464.3-.064.612.015.846.212.163.156.376.252.602.27.262.024.524-.058.726-.227.18-.15.3-.36.339-.588l.011-.087v-.045l.01-.078c.09-.51.555-.878 1.082-.839.053.005.106.014.158.027l.077.024.086.02.087.01c.544.034 1.018-.365 1.077-.905.003-.073 0-.147-.01-.219l-.02-.107-.011-.093c-.003-.042-.002-.084.002-.125l.008-.062c.033-.387.29-.718.658-.845.037-.007.074-.016.11-.027l.054-.02.075-.03c.318-.149.536-.456.567-.81.012-.134-.005-.27-.05-.396.222-.57.382-1.163.479-1.767.05.438.07.878.056 1.319 0 4.788-3.89 8.67-8.689 8.67-4.798 0-8.688-3.882-8.688-8.67-.001-.452.035-.903.107-1.35zm7.921 5.072c.297-.297.75-.377 1.13-.198.38.18.607.578.566.995-.01.116-.04.228-.092.331-.15-.321-.462-.537-.816-.565-.414-.038-.806.189-.98.565-.181-.378-.104-.83.192-1.128zm-2.187-.633l.005.025c.392.024.736.27.884.633.148.362.074.778-.19 1.068l.002-.033-.002-.033c0-.563-.457-1.019-1.02-1.019-.261.003-.511.105-.699.285.045-.5.447-.888.942-.923l.078-.003zm5.653 0c.524-.001.958.405.99.926-.284-.283-.712-.366-1.082-.21-.37.154-.61.517-.607.918v.066c-.187-.188-.293-.442-.295-.707.013-.543.45-.98.994-.993zm-7.89-1.509l.084.007c.533.096.897.59.832 1.125-.038.277-.194.524-.429.677.03-.075.053-.154.067-.234.056-.386-.121-.77-.452-.977-.331-.208-.754-.202-1.079.015.169-.389.559-.63.977-.613zm11.207.613c-.325-.217-.748-.223-1.079-.015-.33.208-.508.59-.452.977.011.08.034.16.066.234-.343-.226-.51-.64-.42-1.042.091-.4.421-.703.829-.76h-.005c.449-.055.88.191 1.06.606zm-6.81-1.383c.347-.236.803-.236 1.15 0 .346.236.513.66.42 1.067.008.068.008.136 0 .204-.217-.322-.581-.513-.97-.51-.397-.012-.773.18-.994.51-.017-.067-.025-.135-.026-.204-.093-.408.074-.831.42-1.067zm-2.371-.452c.366-.025.716.153.91.463.196.31.203.703.02 1.02-.03.063-.067.12-.111.174-.054-.383-.295-.713-.643-.881-.342-.181-.75-.181-1.092 0 .019-.067.044-.132.076-.193.151-.334.474-.559.84-.583zm5.948 0c.367.024.69.249.84.583.03.062.055.127.076.193-.34-.181-.75-.181-1.091 0-.349.168-.59.498-.643.88-.045-.052-.082-.11-.113-.172-.182-.318-.174-.71.02-1.02.195-.311.545-.49.911-.464zm4.064-.782c.315 0 .61.15.796.402-.154-.091-.33-.137-.51-.132-.54.024-.967.464-.974 1.003.002.213.072.42.199.59-.393-.225-.584-.687-.464-1.123.12-.436.52-.736.974-.73l-.02-.01zm-13.602 0c.453-.006.854.294.974.73s-.07.898-.463 1.123c.126-.171.196-.378.199-.59-.014-.553-.467-.993-1.02-.993-.18-.005-.357.04-.511.132.169-.23.429-.374.71-.398l.111-.004zm6.582-1.262c.55 0 .995.444.995.992.008.068.008.136 0 .204-.2-.297-.526-.483-.88-.507l-.09-.002c-.405-.02-.792.173-1.02.509l-.004-.102.004-.102c0-.548.446-.992.995-.992zm-6.99-1.171l.005.025c.453-.006.854.294.974.73s-.07.898-.464 1.123c.128-.17.196-.378.194-.59 0-.553-.441-1.005-.995-1.018-.178-.002-.354.044-.51.132.17-.23.43-.374.711-.399l.085-.003zm13.951.002l.066-.002h-.007c.315 0 .61.149.796.402-.408-.236-.926-.153-1.238.2-.313.352-.332.876-.047 1.25-.393-.225-.584-.687-.464-1.123.113-.41.475-.7.894-.727z"
          fill="#B0AEC2"
        />
        <path
          d="M54.53 47.025v.092c-.096 0-.158.025-.178.05-.025.022-.038.054-.036.087v2.856l2.648-.02c.036.001.07-.014.092-.04.027-.055.043-.114.046-.174h.092v1.319h-.092c0-.053-.016-.104-.046-.148-.03-.01-.062-.01-.092 0h-4.01v-.092c.097 0 .158-.025.179-.05.024-.022.037-.054.035-.087v-3.564c.002-.033-.01-.065-.035-.086l-.04-.017c-.033-.013-.084-.028-.139-.034v-.092h1.577zm-17.754-.046c.095.005.19.015.285.03l.28.057.348.091c.08.022.16.048.237.079l.115.049c.064.033.133.057.204.071.044-.002.084-.024.107-.061l.092.03-.51 1.019-.092-.056c.004-.027.004-.055 0-.082 0-.035-.046-.076-.133-.117-.104-.053-.213-.094-.326-.122-.148-.039-.298-.07-.45-.092-.168-.025-.339-.037-.51-.035-.172 0-.344.025-.51.07-.16.046-.311.122-.444.225-.131.102-.238.232-.31.382-.077.159-.115.333-.113.509-.004.187.043.372.138.534.088.146.206.27.347.367.142.098.301.17.469.214.166.045.338.069.51.071.193 0 .386-.013.577-.04.172-.025.342-.06.51-.108v-.509l-.944.005c-.034-.003-.067.015-.082.046-.017.033-.028.07-.03.107h-.092v-1.069h.092c0 .092.046.137.112.137h2.26v.092c-.035-.002-.068.012-.092.038-.016.017-.026.038-.03.06v1.322c-.006.03-.006.062 0 .092 0 .02.023.031.053.034l.06.002v.091c-.765.383-1.611.572-2.465.55-.314.003-.626-.036-.929-.117-.285-.071-.554-.198-.79-.372-.232-.166-.423-.384-.557-.636-.128-.244-.2-.514-.209-.788v-.118c-.01-.313.063-.624.21-.901.133-.254.323-.473.555-.642.237-.177.505-.307.791-.381.302-.087.615-.13.929-.128h.337zm8.984-.02c.313-.001.624.043.924.132.274.083.53.216.755.392.215.174.39.393.51.642.251.564.251 1.207 0 1.771-.12.249-.295.468-.51.642-.224.177-.48.306-.755.382-.605.162-1.242.162-1.847 0-.275-.077-.531-.206-.755-.382-.216-.174-.39-.393-.51-.642-.26-.562-.26-1.209 0-1.771.12-.25.294-.468.51-.642.225-.175.481-.308.755-.392.3-.089.61-.133.923-.132zm20.311-.163v1.329l-.112-.005c-.003-.053-.019-.103-.046-.148-.017-.02-.04-.032-.066-.035h-2.551v.676h1.699c.034.004.069-.01.092-.035.028-.04.043-.088.046-.138h.091v1.283h-.091c-.002-.059-.018-.116-.046-.168-.023-.027-.057-.042-.092-.04h-1.7v1.288c0 .031.01.062.032.086.055.034.118.052.183.051v.092h-1.607v-.092c.063 0 .125-.017.179-.05.03-.02.05-.052.056-.087v-3.564c0-.033-.013-.064-.036-.087-.053-.034-.115-.051-.179-.05v-.092h3.919c.035.001.07-.014.092-.04.027-.055.043-.114.046-.174h.091zm-20.31 1.094c-.164 0-.326.026-.48.077-.145.048-.28.122-.398.219-.116.096-.21.216-.276.351-.07.148-.104.31-.102.474-.002.165.033.328.102.478.066.135.16.255.276.351.118.097.253.17.398.22.311.102.647.102.959 0 .145-.05.28-.123.398-.22.116-.096.21-.216.275-.35.07-.15.105-.314.102-.48.003-.163-.032-.325-.102-.473-.066-.135-.16-.255-.275-.351-.119-.097-.253-.17-.398-.22-.155-.05-.317-.076-.48-.076zm-16.97.367l.056.03v1.396l-.051.03-3.827-.697v-.077l3.822-.682zm42.377-.03l3.822.697V49l-3.827.698-.05-.03v-1.41l.055-.031zM17.153 24.05c.801.427 1.33 1.163.879 3.12l-.756 3.075c-.393 1.584-1.118 2.704-2.618 3.274 1.163.558 1.376 1.628 1.03 3.136l-.933 3.772c-.51 2.072-1.53 2.836-2.724 3.274H4.357L10.52 24.05h6.633zm72.087 0l1.3 4.073h-3.724l1.02 3.563h3.603l.497 1.584.794 2.489h-3.715l1.102 3.823h4.164l1.3 4.073h-8.209L82.24 24.05h7zm-65.398 0L20.929 38.14h3.525l-.673 3.707h-7.143L20.78 24.05h3.06zm10.872 0c.801.397 1.383 1.09 1.179 2.968-.422 3.957-.845 7.911-1.27 11.862-.205 1.878-1.021 2.545-2.041 2.968h-5.138l-.144-.068c-.824-.408-1.37-1.123-1.07-2.9l1.99-11.862c.316-1.879 1.091-2.546 2-2.968h4.494zm4.99 0H44.2l.145.073c.759.41 1.316 1.127 1.263 2.895l-.138 4.302h-3.23l.215-3.808-.184-.25h-.984l-.215.25-.73 10.874.2.25h1.071l.23-.25.209-3.757h3.337l-.029.6-.027.656c-.027.727-.042 1.346-.069 2.117l-.033.878c-.062 1.878-.827 2.545-1.812 2.968h-5.137l-.158-.072c-.85-.408-1.445-1.125-1.281-2.896.36-3.954.724-7.908 1.092-11.862.168-1.879.893-2.546 1.77-2.968zm11.04-.02l.004 1.45.01 1.34c.013 1.28.033 2.452.048 4.133h.403l.32-1.39.366-1.617c.266-1.198.509-2.33.845-3.917h2.806l.122 2.673-.32 1.353c-.397 1.66-.696 2.847-1.21 4.822l.691 2.1c.447 1.34.836 2.468 1.441 4.183l.123 2.667h-3.204l-.566-2.002-.359-1.294c-.321-1.174-.621-2.31-1.024-3.872l-.388.02.066 7.169h-3.515l.28-17.819h3.062zm9.48.02l1.194 14.091h3.531l.408 3.707h-7.143l-1.05-17.798h3.06zm11.225 0l.648 3.706H68.73l.44 2.95.04.267h3.244l.643 3.707H69.76l.51 3.461h3.74l.653 3.707h-7.352L65.107 24.05h6.342zm9.189 0l.923 3.706h-2.275c1.081 4.697 2.165 9.394 3.25 14.092H79.02c-.966-4.698-1.933-9.395-2.903-14.092h-2.275l-.7-3.706h7.496zM12.02 35.515h-1.387l-1.409 4.912h1.44l.316-.275 1.199-4.368-.115-.197-.044-.072zm20.526-8.253h-1.02l-.116.127-.075.078-.044.045-1.53 10.874.178.25h1.076l.245-.25 1.424-10.874-.138-.25zm-18.26.061H12.98l-1.409 4.918h1.358l.3-.275 1.2-4.368-.143-.275z"
          fill="#FFF"
        />
        <path
          d="M15.78 36.075c-.022.162-.05.323-.083.483l-.054.24-.898 3.63c-.5 2.01-1.471 2.789-2.63 3.243l-.105.04H4.357l1.617-5.157c1.073-.337 2.168-.656 3.285-.958l.843-.223-.878 3.054h1.44l.316-.275.27-.966c.225-.813.397-1.465.617-2.256 1.286-.306 2.59-.59 3.914-.855zm69.664.315c1.36.292 2.695.608 4.005.947l.653 2.266h4.179l1.321 4.098h-8.23c-.632-2.437-1.275-4.874-1.928-7.31zm-63.867-1.303l-.633 3.054h3.525l-.234 1.282c-.07.386-.133.747-.201 1.138l-.228 1.312h-7.143l1.434-6.21c.869-.157 1.74-.302 2.61-.44l.87-.136zm25.964-2.138h2.762l.806.004c.959.007 1.983.023 3.054.052l.703 2.134c.426 1.278.81 2.388 1.394 4.041l.123 2.668h-3.204l-.567-2.002c-.422-1.51-.75-2.728-1.197-4.45l-.185-.716h-.378l.053 4.865c.003.487.006 1.001.007 1.57l.001.712h-3.51l.138-8.878zm-12.337.58c-.194 1.785-.386 3.569-.576 5.35-.203 1.822-.972 2.528-1.95 2.932l-.091.037h-5.143l-.147-.07c-.794-.396-1.333-1.08-1.087-2.716l.79-4.734c1.136-.135 2.264-.26 3.383-.371l-.638 4.434.178.25h1.082l.245-.25.607-4.582c1.133-.102 2.248-.195 3.347-.28zm5.46-.336l-.348 5.193.098.125.101.124h1.072l.23-.25.209-3.756h3.336l-.132 4.25c-.06 1.824-.783 2.53-1.725 2.933l-.087.036h-5.137l-.158-.072c-.825-.396-1.409-1.082-1.299-2.738l.013-.158.51-5.499c1.14-.074 2.245-.137 3.316-.188zm17.05-.071c.857.038 1.735.081 2.634.13l.678.038.413 4.851h3.53l.409 3.707h-7.143c-.177-2.909-.35-5.818-.52-8.726zm8.613.54c2.193.183 4.454.412 6.744.692l.036.159.015.081.006.086H69.79l.318 2.18.192 1.281h3.714l.653 3.707h-7.357c-.337-2.73-.665-5.458-.984-8.187zm11.28 1.303c.773.115 1.546.237 2.318.366l1.157.199 1.464 6.318-3.526-.01-1.413-6.873z"
          fill="url(#xdra1xeu7b)"
        />
        <path
          d="M95.102 53.317v.728h-.138v-.728H94.7v-.112h.663v.143l-.26-.03zm1.214.739v-.708l-.285.697h-.087l-.27-.702v.697h-.138v-.84h.214l.245.642.25-.637h.214v.84l-.143.01z"
          fill="#262261"
        />
      </g>
    </svg>
  </a>
</h1>
`;
